src/auth/LoginPage.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Login, useTranslate } from 'react-admin';
+import { Button, Card, CardContent } from '@material-ui/core';
+import { ThemeProvider } from '@material-ui/styles';
+import { makeStyles } from '@material-ui/core/styles';
+
+import { loginTunnistamo } from './authenticate';
+import theme from '../theme';
+
+const useStyles = makeStyles({
```

Comment:
Things i hate the most in my entire dev life :) Inline stylings.

React file is mixed between templating and logic, now styles..

Could you extract this config in a file called MaterialUIConfigs.ts?